### PR TITLE
SC-305 Create a new interface for calendar feed administration

### PR DESCRIPTION
**Pull request by CeEv**
Depends on SC-305 in schulcloud-server and schulcloud-calendar.

### CODE REVIEW

**Comment by schurigh** [comment]
still WIP? how's the status?

**Comment by CeEv** [comment]
Merge master and scope change is open.

**Comment by janrenz** [comment]
should we reactivate this?

**Comment by CeEv** [comment]
> should we reactivate this?

If i have a free time slot in next weeks, i will make i done.

**Comment by adrianjost** [comment]
@CeEv how's the status? Is it ready to merge?

**Comment by janrenz** [comment]
@CeEv Are you ok if we close this, if we want to intorduce this this should be part of "Datenquellen" and implemented in Vue

File: controllers/administration.js [hunk 1]
```diff
@@ -1832,4 +1833,121 @@ router.all('/systems', function (req, res, next) {
...
+			return [		
+				attributes['description'],
+				attributes['ics-url'],
+				attributes['target'],	//!todo: target is a mapping from scopeId
+				getTableActions(
+					attributes,
+					'/administration/calendar/',
+					_.includes(res.locals.currentUser.permissions, 'ADMIN_VIEW')
+				)
+			]
```
Comment (tofixx) [inline controllers/administration.js:1938]: Codacy found an issue: [Missing semicolon.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-client/pullRequest?prid=6045542)

**Comment by tofixx** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Clones added
============
- controllers/administration.js  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-client/pullRequest?prid=6045542&bid=19807425)

**Pull request closed without merging**